### ISSUE TITLE
Can't build Servo on macOS 10.15

### ISSUE THREAD

**Issue by jdm** [servo/servo#25919]
From the spidermonkey upgrade, I now get this build failure:
```
--- stderr
ERROR: SDK version "10.15" is unsupported. Please downgrade to version 10.14. You may need to point to it using --with-macos-sdk=<path> in your mozconfig. Various SDK versions are available from https://github.com/phracker/MacOSX-SDKs
make: *** [maybe-configure] Error 1
thread 'main' panicked at 'assertion failed: result.success()', /Users/joshmatthews/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/aabcc9b/build.rs:181:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
We're going to need to add some build configuration to support this to mozjs, and possibly make downloading the 10.14 SDK part of mach bootstrap on macos.

**Comment by nox** [comment]
We should just backport [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1583854).

**Comment by nox** [comment]
https://github.com/servo/mozjs/pull/234

**Comment by jdm** [comment]
This hasn't landed in master yet, so I'd prefer to keep this open.

**Comment by nox** [comment]
Oops, sorry about that;

**Comment by CYBAI** [comment]
Maybe this is fixed by #25946?

**Comment by nox** [comment]
It is.
